restricted_site_access.php

```diff
@@ -87,6 +90,35 @@ public static function ajax_notice_dismiss() {
 		wp_send_json_success();
 	}
 
+	/**
+	 * Keeps log of network-wide RSA disable action
+	 *
+	 * @return void
+	 */
+	public static function ajax_network_disable() {
+		if ( ! check_ajax_referer( 'rsa_admin_nonce', 'nonce', false ) ) {
+			wp_send_json_error();
```

Comment:
can we send a meaningful error message, even if its vague. eg 'Error: action not allowed'